app/src/main/java/com/newgogiyumandroid/MainActivity.kt

```diff
@@ -1,15 +1,26 @@
 package com.newgogiyumandroid
 
-
-import androidx.appcompat.app.AppCompatActivity
+import android.net.ConnectivityManager
+import android.net.NetworkInfo
 import android.os.Bundle
+import android.view.View
+import androidx.appcompat.app.AppCompatActivity
 import androidx.recyclerview.widget.LinearLayoutManager
 import com.newgogiyumandroid.JsonParsingLists.FoodList
 import com.newgogiyumandroid.RecyclerView.CustomAdapter
 import kotlinx.android.synthetic.main.activity_main.*
+import kotlinx.android.synthetic.main.activity_main.view.*
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import okhttp3.*
 import org.json.JSONArray
+import java.io.IOException
 import java.net.URL
 
+// SplashActivity 추가도 고려해볼만 하다
```

Comment:
Can you remove this comment, and add formal class style comment?

Reply:
Please take a look 
https://www.tutorialspoint.com/java/java_documentation.htm

Reply:
Saw removed comment. 
I think we don't need comment yet for MainActivity.

Reply:
You can add class style comment next time, but you don't need for now.